pyqir-jit/pyqir/jit/nonadaptivejit.py

```diff
@@ -16,13 +16,18 @@ class NonadaptiveJit(object):
     def __init__(self):
         self.jit = PyNonadaptiveJit()
 
-    def eval(self, file_path: str, gateset: GateSet, entry_point: Optional[str] = None):
+    def eval(self,
+             file_path: str,
+             gateset: GateSet,
+             entry_point: Optional[str] = None,
+             result_stream: Optional[List[bool]] = None):
```

Comment:
Should `result_stream` be required? It may be less surprising for pyqir-jit to panic if a measurement is made past the end of the result stream, rather than silently return zero.

Reply:
Updated the documentation for `NonadaptiveJit` to explicitly state this behavior.